src/components/LoadingSpinner/LoadingSpinner.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import styled from "styled-components";
+
+import LoadingSpinner from "./LoadingSpinner";
+
+export default {
+  title: "LoadingSpinner",
+  component: LoadingSpinner,
+  args: {},
+} as ComponentMeta<typeof LoadingSpinner>;
+
+const Template: ComponentStory<typeof LoadingSpinner> = () => (
```

Comment:
This is pretty lazy story as this component has a single property which is for size, so I figured just rendering them in one place is enough. Let me know if you wanna separate them